api/capi/include/tensor_filter_single.h

```diff
@@ -30,6 +30,7 @@
 
 #include <nnstreamer/nnstreamer_subplugin.h>
 #include <nnstreamer/nnstreamer_plugin_api_filter.h>
+#include <nnstreamer/tensor_filter/tensor_filter_common.h>
```

Comment:
Does this really need to include tensor_filter_common.h?
It'd be better if we can include tensor_filter_common.h in *.c only.

Reply:
ah.. this is a private header. please never mind.